src/actions/seo-generate-cards.ts

```diff
@@ -81,7 +81,7 @@ export async function action(issueId?: string) {
     }
 
   } catch (err) {
-    logger.error(`${name} Error`, err?.message || err.toString());
+    logger.error(`${name} Error`, err);
```

Comment:
Please, don't change the error logging as that will lead to problems with elastic-search

Reply:
This problem was blocking from running the deploy on jenkins. but before merging I will remove